Description: Remove LD_LIBRARY_PATH from envvars-std
Forwarded: no
Author: Adam Conrad <adconrad@0c3.net>
Last-Update: 2012-02-25
Index: apache2/support/envvars-std.in
===================================================================
--- apache2.orig/support/envvars-std.in
+++ apache2/support/envvars-std.in
@@ -18,7 +18,4 @@
 #
 # This file is generated from envvars-std.in
 #
-@SHLIBPATH_VAR@="@exp_libdir@:$@SHLIBPATH_VAR@"
-export @SHLIBPATH_VAR@
-#
 @OS_SPECIFIC_VARS@
